smarts/core/sensors.py

```diff
@@ -503,9 +508,12 @@ def _vehicle_is_off_route_and_wrong_way(cls, sim, vehicle):
             # Lanes from an edge are parallel so any lane from the edge will do for direction check
             # but the innermost lane will be the last lane in the edge and usually the closest.
             lane_to_check = route_edge_or_oncoming.getLanes()[-1]
-            is_wrong_way = cls._vehicle_is_wrong_way(
-                sim, vehicle, lane_to_check.getID()
-            )
+            if lane_to_check.getEdge().isSpecial():
+                is_wrong_way = False
```

Comment:
You should probably add a comment here (referencing the Issue).
(... and in the other spot you did this.)

Reply:
ok.